### DISCUSSION TITLE
Automatically signout user when session expire?

### DISCUSSION THREAD

**Discussion by meerbahadin** [nextauthjs/next-auth#1994] [Help]
Hello there,

I was just wondering is there's a proper way to listen for session/JWT existing r expire time and logout user automatically for examples when you open a github in two different tab and login in one of them the other one warn you "You signed in with another tab or window. Reload to refresh your session"

**Comment by balazsorban44** [comment]
We have tab syncing, so if you switch between tabs, you should automatically be logged in in both. If you use JWT for a session, it is saved in a cookie, and when it expires, you will be "logged out", meaning the cookie will just disappear.
